test/api_test.js

```diff
@@ -46,55 +46,35 @@ suite('api', () => {
     });
   });
 
-  test('all builds', async function(done) {
+  test('all builds', async function() {
     let builds = await helper.github.builds();
-    try {
-      assert.equal(builds.builds.length, 4);
-      assert.equal(builds.builds[0].organization, 'abc123');
-      assert.equal(builds.builds[1].organization, 'ghi789');
-      done();
-    } catch (e) {
-      done(e);
-    }
+    assert.equal(builds.builds.length, 4);
```

Comment:
iirc, if line 50 `await helper.github.builds();` throws an exception, this test will hang rather than complete now? I might be mis-remembering though.